src/pages/settings/Privacy.tsx

```diff
@@ -72,7 +72,7 @@ export const Privacy: SettingsStackScreen<"Privacy"> = () => {
   }, [loggedIn])
 
   return (
-    <ContentWrapperScroll title="Privacy">
+    <ScrollPage upperTitle="Privacy">
       <BodyText
         style={{
           paddingTop: 36,
```

Comment:
Togliendo overflow:"hidden" dalla ScrollPage come avevo scritto nell'altro commento, i rettangoli grigi della pagina di privacy fuoriescono durante lo scrolling.

Immagino vada usata una View che wrappa la ScrollView con overflow:"hidden"?

Mi sa che questo problema lo avevamo già avuto in passato right?

Reply:
sì fixato wrappando la scrollview in una view